include/onnxruntime/core/framework/data_types.h

```diff
@@ -34,6 +34,13 @@ using MapInt64ToInt64 = std::map<int64_t, int64_t>;
 using MapInt64ToFloat = std::map<int64_t, float>;
 using MapInt64ToDouble = std::map<int64_t, double>;
 
+//unordered maps
+using UnorderedMapStringToString = std::unordered_map<std::string, std::string>;
```

Comment:
The things above exist there for historical reasons. They are they data types that used in ONNX-ML operators. UnorderedMapStringToString isn't one. So, if you put it here, it could cause unnecessary confusion.

Reply:
I've removed the alias from this header file.